src/traces/choropleth/attributes.js

```diff
@@ -40,7 +41,7 @@ module.exports = extendFlat({
     }),
     marker: {
         line: {
-            color: scatterGeoMarkerLineAttrs.color,
+            color: extendFlat({}, scatterGeoMarkerLineAttrs.color, {dflt: defaultLine}),
```

Comment:
Changing the default here changes a lot of baselines! How did you choose the new color? Also, should we change `scattergeo`'s default as well?

Reply:
There was a bug in the previous setup not picking the right default color. The new baselines are OK.

Reply:
_discussed in-person with @antoinerg_

Resolved.